knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -191,22 +246,32 @@ func installKnativeKafkaChannel(instance *operatorv1alpha1.KnativeKafka, apiclie
 	return nil
 }
 
+// rawKafkaChannelManifest returns KafkaChannel manifest without transformations
+func rawKafkaChannelManifest(apiclient client.Client) (mf.Manifest, error) {
+	return mfc.NewManifest(kafkaChannelManifestPath(), apiclient, mf.UseLogger(log.WithName("mf")))
+}
+
 func kafkaChannelManifest(instance *operatorv1alpha1.KnativeKafka, apiClient client.Client) (*mf.Manifest, error) {
-	manifest, err := mfc.NewManifest(kafkaChannelManifestPath(), apiClient, mf.UseLogger(log.WithName("mf")))
+	manifest, err := rawKafkaChannelManifest(apiClient)
 	if err != nil {
 		return nil, fmt.Errorf("failed to load KafkaChannel manifest: %w", err)
 	}
 
-	manifest, err = manifest.Transform(mf.InjectOwner(instance))
```

Comment:
Why not keep this format and just keep adding the transformers directly? It takes a variadic argument anyway.

Reply:
Yep